src/templates/token.tsx

```diff
@@ -200,6 +285,23 @@ const generateTokenPage = ({ secret }: { secret: string }) => {
               >
                 <span className="break-all">{secret}</span>
               </MjmlText>
+              <MjmlSpacer height={values.px16} />
+              <MjmlText
+                mjClass="font-accent text-formal-accent"
+                fontWeight={400}
+                fontSize={values.px14}
+                lineHeight={values.px22}
+              >
+                <span className="break-all">
+                  Use this token here:{' '}
+                  <a
+                    className="link-text"
+                    href="https://sealc.red/app?email={domain}"
```

Comment:
Here you can use your `const` link as well

Reply:
Changed